spec/support/osx-spec.js

```diff
@@ -248,9 +262,19 @@ describe('StateController - OSX Support', () => {
               onMount: jasmine.createSpy(),
               onCopy: jasmine.createSpy(),
               onUnmount: jasmine.createSpy(),
-              onRemove: jasmine.createSpy(),
+              onRemove: () => {
```

Comment:
Nice way to handle this, it could also have been: 

```js
onRemove: jasmine.createSpy().andCallFake(() => {
  ...
}),
```

So that we can still have a test for the `onRemove` call.